packages/mobile/src/components/TotalLineItem.tsx

```diff
@@ -38,15 +40,14 @@ export default function TotalLineItem({ title, amount, hideSign, currencyInfo }:
         textStyle={fontStyles.regular600}
         amount={<CurrencyDisplay amount={amount} hideSign={hideSign} currencyInfo={currencyInfo} />}
       />
-      {localCurrencyCode !== LocalCurrencyCode.USD && exchangeRate && (
+      {exchangeRate && (
         <LineItemRow
           title={
-            <Trans i18nKey="totalInDollars" ns={Namespaces.global}>
-              Celo Dollars @{' '}
+            <Trans i18nKey={totalAmountKey[txCurrency]} ns={Namespaces.global}>
               <CurrencyDisplay
                 amount={{
-                  value: new BigNumber(exchangeRate).pow(-1),
-                  currencyCode: CURRENCIES[CURRENCY_ENUM.DOLLAR].code,
+                  value: new BigNumber(exchangeRate).pow(txCurrency === Currency.Celo ? 1 : -1),
+                  currencyCode: CURRENCIES[Currency.Dollar].code,
```

Comment:
Added a comment, but the currency used here is actually the local currency, using Dollar just so it shows up as `$`. It would be ideal to show the local currency sign, but that would require extra work. This part is not working very well in production today.